packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
I am not saying that we can use repository without a model :) I was wondering if `Model` would be one of the refactored blocks to its own repo .. If not Model, then maybe the other stuff .. 
I agree about taking the explanation details a bit below.. I would create a new section and call it `Concept` and move it under `Related Source`, if everybody is ok with that.